prow/cmd/deck/BUILD

```diff
@@ -1,5 +1,29 @@
 package(default_visibility = ["//visibility:public"])
 
+load("@io_bazel_rules_docker//container:container.bzl", "container_image")
+
+container_image(
+    name = "asset-base",
+    base = "@alpine-base//image",
+    # With paths relative to the current directory.
+    data_path = ".",
+    # Add files under into the root directory.
+    directory = "/",
+    # Everything under static/
+    files = glob(["static/**/*"]),
+)
+
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
```

Comment:
nit: can we keep all of the `load()s` at the top of the file?

Reply:
Done